### PR TITLE
richedit - table wrapping

### PR DESCRIPTION
**Pull request by geumjaLee**

### CODE REVIEW

File: interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+* Place the cursor within the target table.
+* Click the **Properties** button in the **Table** group on the **Table Tools/Layout** [tab](../text-editor-ui/ribbon-interface.md) or invoke the context menu and select **Table Properties**.
+
+    ![RTETablesPropertiesRibbon](../../../images/img121442.png)
+
+    ![RTETablesPropertiesContextMenu](../../../images/img121443.png)
+* In the invoked **Table Properties** dialog, click **Around** in the **Text Wrapping** section. Click the enabled **Positioning** button to open the **Table Positioning** dialog.
+
+    ![RTETablesPropertiesRibbon](../../../images/richedit-tables-wrapping-dialog.png)
+* Here you can specify the table's horizontal and vertical position. Enter numeric value in the **Position** combo box to specify an absolute position. To set a relative position, select the required alignment from the **Position** list and the relative element from the **Relative to:** list.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md:12]: I'd like to make it more specific than "Here". Can we update it as follows?

```suggestion
* In this dialog, you can specify the table's horizontal and vertical positions. Enter a numeric value in the **Position** combo box to specify an absolute position. To set a relative position, select the required alignment from the **Position** list and the relative element from the **Relative to:** list.
```

File: interface-elements-for-desktop/articles/rich-text-editor/footnotes-and-endnotes/change-note-appearance.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+To specify what note options to change, click the **Footnotes** or **Endnotes** radio button in the dialog's **Location** section.
+
+You can apply changes to the current section or to the whole document. Select the required options in the **Apply changes to:** drop-down list.
+
+![apply to](../../../images/rich-edit-footnotes-and-endnotes-apply-to.png)
+
+## Change Note Location
+
+Select the desired note location from the **Location** drop down menu. You can place footnotes at the bottom of the page or below text. Endnotes can be located at the end of the document or at the end of each section.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/rich-text-editor/footnotes-and-endnotes/change-note-appearance.md:21]:
```suggestion
Select the desired note location from the **Location** drop-down menu. You can place footnotes at the bottom of the page or below text. Endnotes can be located at the end of the document or at the end of each section.
```

File: interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+* Specify the distance between the table and surrounding text in the **Distance from surrounding text** section.
+
+* Check **Move with Text** to make the table move with surrounding text.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md:16]: Can you clarify what you mean by "make the table move with surrounding text" - for my understanding?

Comment (geumjaLee) [reply interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md:16]: the table can be fixed - when new text is added or removed around the table, the table stays still. When you check this box, the table will move depending on the added or removed text

Comment (albertov05) [reply interface-elements-for-desktop/articles/rich-text-editor/tables/wrap-text-around-a-table.md:16]: Okay, thank you.

**Pull request merged**